ui/src/asm_cleanup.rs

```diff
@@ -4,6 +4,7 @@
 use regex::Regex;
 use regex::Captures;
 use rustc_demangle::demangle;
+use std::collections::HashMap;
```

Comment:
FWIW, you don't have to change this in this PR, but I'll probably rewrite these as `std::collections::{HashMap, HashSet}` at a later point;

Reply:
I'll condense the collections and regex imports.